### PR TITLE
feat: [CDS-36176]: delgate selector addition in advance tab and pipel…

### PR DESCRIPTION
**Pull request by dhruvUpadhyay07**
…ine level

##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by dhruvUpadhyay07** [comment]
dockerize

File: src/modules/10-common/components/MultiTypeDelegateSelector/MultiTypeDelegateSelector.tsx [hunk 1]
```diff
@@ -31,6 +31,7 @@ export interface MultiTypeDelegateSelectorProps extends IFormGroupProps {
   allowableTypes?: MultiTypeInputType[]
   tooltipProps?: DataTooltipInterface
   inputProps: Omit<DelegateSelectorsProps, 'onChange'>
+  formik?: any
```
Comment (diptiman-adak) [inline src/modules/10-common/components/MultiTypeDelegateSelector/MultiTypeDelegateSelector.tsx:34]: Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9248504)
```suggestion
  formik?: unknown
```

Comment (diptiman-adak) [inline src/modules/10-common/components/MultiTypeDelegateSelector/MultiTypeDelegateSelector.tsx:34]: Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9248504)
```suggestion
  formik?: unknown
```

**Comment by dhruvUpadhyay07** [comment]
retrigger jest

**Comment by dhruvUpadhyay07** [comment]
retrigger jest

**Comment by dhruvUpadhyay07** [comment]
dockerize

**Comment by dhruvUpadhyay07** [comment]
dockerize

**Pull request closed without merging**